yarn-project/types/src/interfaces/state_provider.ts

```diff
@@ -24,11 +24,11 @@ export interface StateInfoProvider {
   getContractPath(leafIndex: bigint): Promise<SiblingPath<typeof CONTRACT_TREE_HEIGHT>>;
 
   /**
-   * Returns the sibling path for the given index in the data tree.
+   * Returns the sibling path for the given index in the note hash tree.
    * @param leafIndex - The index of the leaf for which the sibling path is required.
    * @returns The sibling path for the leaf index.
    */
-  getDataTreePath(leafIndex: bigint): Promise<SiblingPath<typeof NOTE_HASH_TREE_HEIGHT>>;
+  getNoteHashTreePath(leafIndex: bigint): Promise<SiblingPath<typeof NOTE_HASH_TREE_HEIGHT>>;
```

Comment:
See comment from earlier.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/3117/commits/13b3f90ebaf71a91bf3fe0d7068135515dfd2078